src/reactive.coffee

```diff
@@ -343,13 +343,62 @@ rxFactory = (_, $) ->
     splice: (index, count, additions...) -> @spliceArray(index, count, additions)
     insert: (x, index) -> @splice(index, 0, x)
     remove: (x) ->
-      i = _(@raw()).indexOf(x)
+      i = _(rx.snap => @all()).indexOf(x)
```

Comment:
`raw` already calls all. I'm not sure who made that change, but there seems to be no reason for its continued existence.